internal/models/kubernetessecret/cluster/secretexport/secret_export.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package secret
+
+import (
+	"github.com/go-openapi/swag"
+
+	conditions "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/kubernetessecret/cluster"
+	objectmetamodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/objectmeta"
+)
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportSecretExport Represents Tanzu Secret Export.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.secretexport.SecretExport
+type VmwareTanzuManageV1alpha1ClusterNamespaceSecretexportSecretExport struct {
```

Comment:
`SecretexportSecretExport` can be `SecretExport`, same applied for the secrets.

Reply:
Updated PTAL

Reply:
@Shubbhang351 this is a nit, you can keep it same as well. 

The reason being in the API model we get the full path to the request (auto generated via swagger gen)